src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizePagesFormatter.java

```diff
@@ -7,6 +7,8 @@
 import net.sf.jabref.logic.formatter.Formatter;
 import net.sf.jabref.logic.l10n.Localization;
 
+import com.mysql.jdbc.StringUtils;
```

Comment:
Good question :smile: I was just noticing this proprietary import that seemed like strongly mysql-related too me.